manifests/cf-manifest/operations.d/310-router.yml

```diff
@@ -26,6 +34,24 @@
         - name: Strict-Transport-Security
           value: max-age=31536000; includeSubDomains; preload
 
+- type: replace
+  path: /instance_groups/name=router/jobs/name=gorouter/properties/router/max_idle_connections?
+  # See: https://docs.cloudfoundry.org/adminguide/routing-keepalive.html
+  #
+  # Do not change this until routing-release >= 0.193
```

Comment:
This comment doesn't make perfect sense now that you're pinning routing release `v=0.193.0`

Reply:
Resolved in 14c1c0a2a313ce5a16faf745fd71b6856696d2d8